IPs/RSKIP176.md

```diff
@@ -0,0 +1,130 @@
+# Fast Bridge
+|RSKIP          |           |
+| :------------ |:-------------|
+|**Title**      |Fast Bridge |
+|**Created**    |15-SEP-20 |
+|**Author**     |GM |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP allows a user to transfer BTC to RSK in a fast way, where a third party takes the risk to advance the payment in RBTC to the user.
+
+## Motivation
+
+Allowing users to transfer BTC to RSK immediately, without waiting for usual peg-in confirmations. This also generates a market between user who want to fast peg-in and those who count on liquidity.
+
+## Specification
+
+The system comprises one or more liquidity providers (LPs), listed in an onchain marketplace. Interactions with LPs are intermediated using a dApp or a website.
+This RSKIP specifies the requirements for consensus changes without limiting the actual implementation of the Liquidity Providers market.
+
+#### Bridge
+
+The bridge is the most affected by this project. It will include:
+- Create **registerBtcTransfer**() method.
+
+#### registerBtcTransfer()
+
+**ABI Signature**
+
+        function registerBtcTransfer(bytes btcTxSerialized, int heigth, bytes pmtSerialized, bytes32 derivationArgumentsHash, string userRefundBtcAddress, string LiquidityBridgeContractAddress, string LiquidityProviderBtcAddress, bool shouldTransferToContract)
```

Comment:
I think this method should return a numeric status with the result of the operation.

Reply:
Forgot to put this